internal/config/debug_test.go

```diff
@@ -22,18 +22,14 @@ import (
 	"testing"
 
 	"github.com/vdaas/vald/internal/errors"
+	"go.uber.org/goleak"
 )
 
 func TestDebug_Bind(t *testing.T) {
+	t.Parallel()
```

Comment:
Should we remove parallel test 🤔 
if other tests are running in parallel, does the bug occurs due to the environment variable?:thinking:

Reply:
could you tell me more why you think?

Reply:
I think environment variable may share with other test.
if we set `ENABLE` to environment variable in this case and other test also set `ENABLE` to environment variable, 
I thought it might be different from the intended result.
If I made a mistake, I am sorry...

Reply:
thx.
But, in these test, setting the environment values are not used.